### PR TITLE
Update default logging subsystem config to be more in line with current code

### PR DESCRIPTION
**Pull request by ctomc**
no need to have mgmt config logger set to debug,
we also don't use jacorb & tomcat/jbossweb anymore.

### CODE REVIEW

File: logging/src/main/resources/subsystem-templates/logging.xml [hunk 1]
```diff
@@ -35,24 +35,12 @@
    <supplement name="default">
       <replacement placeholder="@@append@@" attributeValue="true"/>
       <replacement placeholder="LOGGERS">
-	      <logger category="com.arjuna">
-	          <level name="WARN"/>
-	      </logger>
-	      <logger category="org.apache.tomcat.util.modeler">
-	          <level name="WARN"/>
-	      </logger>
-	      <logger category="org.jboss.as.config">
```
Comment (jamezp) [inline logging/src/main/resources/subsystem-templates/logging.xml]: This one should probably stay.

File: logging/src/main/resources/subsystem-templates/logging.xml [hunk 2]
```diff
@@ -35,24 +35,12 @@
...
-	          <level name="DEBUG"/>
-	      </logger>
-	      <logger category="sun.rmi">
-	          <level name="WARN"/>
-	      </logger>
-	      <logger category="jacorb">
-	          <level name="WARN"/>
-	      </logger>
-	      <logger category="jacorb.config">
```
Comment (jamezp) [inline logging/src/main/resources/subsystem-templates/logging.xml]: For both jacorb ones we need to determine if we should add new categories.

**Comment by jamezp** [comment]
The various logging.properties files also need to be updated too.

**Comment by ctomc** [comment]
will work on this when we move to 2.x

**Pull request closed without merging**